pkg/api/render.go

```diff
@@ -55,6 +57,15 @@ func (hs *HTTPServer) RenderToPng(c *m.ReqContext) {
 		return
 	}
 
+	if err != nil && err == rendering.ErrPhantomJSNotInstalled {
+		if strings.HasPrefix(runtime.GOARCH, "armv") {
+			c.Handle(500, "Rendering failed - PhantomJS isn't included in arm builds per default", err)
```

Comment:
I'd change "included in arm builds" to "included in arm build". Technically we have several arm builds, but for the end user running into this problem only one of them is relevant.